src/moonlink/src/table_handler/chaos_test.rs

```diff
@@ -417,6 +432,13 @@ impl ChaosState {
 
     /// Get a random row to delete.
     fn get_random_row_to_delete(&mut self) -> MoonlinkRow {
+        if self.is_upsert_table && self.rng.random_range(0..100) < 50 {
```

Comment:
The row returned here could be used in both streaming transaction or non-streaming transaction, but you only implement "delete if exists" semantics for non-streaming txn, I think it's breaking chaos test